apps/web/src/utils/css.ts

```diff
@@ -34,32 +34,33 @@ export function injectCssSrc(id, src) {
   head.appendChild(link);
 }
 
-export function injectCss(rule) {
-  let variableCss = document.getElementById("variables");
-  let head = document.getElementsByTagName("head")[0];
+export function injectCss(rule: string) {
+  const variableCss = document.getElementById("variables");
+  const head = document.getElementsByTagName("head")[0];
   if (variableCss) {
     head.removeChild(variableCss);
   }
-  let css = document.createElement("style");
+  const css = document.createElement("style");
   css.type = "text/css";
   css.id = "variables";
   // Support for IE
-  if (css.styleSheet) css.styleSheet.cssText = rule;
+  if ((css as any).styleSheet) (css as any).styleSheet.cssText = rule;
```

Comment:
We don't support IE so you can remove this.

Reply:
Ok